brainwidemap/bwm_loading.py

```diff
@@ -181,15 +197,26 @@ def load_trials_and_mask(one, eid, min_rt=0.08, max_rt=2., nan_exclude='default'
             'feedbackType'
         ]
 
-    sess_loader = SessionLoader(one, eid)
-    sess_loader.load_trials()
+    if sess_loader is None:
+        sess_loader = SessionLoader(one, eid)
+
+    if sess_loader.trials.shape[0] == 0:
+        sess_loader.load_trials()
 
     # Create a mask for trials to exclude
     # Remove trials that are outside the allowed reaction time range
     query = f'(firstMovement_times - stimOn_times < {min_rt}) | (firstMovement_times - stimOn_times > {max_rt})'
+    # Remove trials that are outside the allowed trial duration range
+    query += f' | (feedback_times - goCue_times < {min_trial_len}) | (feedback_times - goCue_times > {max_trial_len})'
```

Comment:
@themattinthehatt This would change the default behavior of this function, can we set these to None by default?

Reply:
now `None` by default